UsefulFlags.md

```diff
@@ -20,6 +20,10 @@ Keep the sandbox directory (`/var/tmp/_bazel_$USER/*/sandbox/darwin-sandbox/*`)
 
 Print jobs run by the `clang` and `swiftc` drivers respectively. For example this will show compiler frontend invocations, linker invocations, etc.
 
+###### `--experimental_generate_json_trace_profile`
+
+Run `bazel build` with this flag to generate a Chrome compatible trace file that can be used to visually introspect the build's timing, sequencing, and parallelism. See chrome://tracing
```

Comment:
I believe you also need --profile=path with this. Otherwise it might be a no op?

Reply:
I was wondering about that, thanks. I'll update.

Reply:
even if it's not an example invocation would be fine and cover it